stack/services/src/main/java/org/apache/usergrid/services/notifications/ProviderAdapterFactory.java

```diff
@@ -37,6 +38,7 @@ public static ProviderAdapter getProviderAdapter(Notifier notifier, EntityManage
        switch(notifier.getProvider().toLowerCase()){
            case "apple" : adapter = new APNsAdapter(entityManager,notifier); break;
            case "google" : adapter = new GCMAdapter(entityManager ,notifier); break;
+           case "windows" : adapter = new WNSAdapter(entityManager ,notifier); break;
            case "noop" : adapter = new TestAdapter(notifier); break;
```

Comment:
Seems silly, but I think our default case should be to throw an exception if it doesn't match any of the other cases.  That way we get some explicit feedback something was entered incorrectly into the payload.